Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs

```diff
@@ -88,6 +88,9 @@ public FileTransferResult Run(CancellationToken cancellationToken)
 
                     using (var client = new SftpClient(connectionInfo))
                     {
+                        client.ConnectionInfo.KeyExchangeAlgorithms.Remove("curve25519-sha256");
```

Comment:
Can you add an explanation in a comment please?

Reply:
fixed